mitmproxy/utils/human.py

```diff
@@ -62,3 +62,9 @@ def format_timestamp(s):
 def format_timestamp_with_milli(s):
     d = datetime.datetime.fromtimestamp(s)
     return d.strftime("%Y-%m-%d %H:%M:%S.%f")[:-3]
+
+
+def format_ip_address(addr, port):
```

Comment:
1. Let's call this `format_address` - after all, it should also work for hostnames.
2. Let's just accept a single `(host, port)` tuple here, not two different arguments. That's not particularly beautiful, but matches all the other Python socket APIs.